command/debug/debug.go

```diff
@@ -413,7 +413,7 @@ func (c *cmd) captureLongRunning(ctx context.Context) error {
 
 		g.Go(func() error {
 			// use ctx without a timeout to allow the trace to finish sending
-			return c.captureTrace(ctx, s)
+			return c.captureTrace(ctx, int(c.interval.Seconds()))
```

Comment:
Unrelated to this change, but I just noticed that the adjustment in L405-407 isn't doing anything given that there's a `debugMinDuration` set to 10s.

Reply:
Ya, I noticed that as well, but decided to leave it for now to keep the PR focused on this fix. I'm happy to remove those lines in another commit.